portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportImport.testcase

```diff
@@ -2004,243 +2351,283 @@ definition {
+			WebContentNavigator.gotoAddWithStructureCP(structureName = "Structure Name B");
 
-		WebContent.addWithStructureCP(
-			structureName = "Structure Name B",
-			templateName = "Template Name B",
-			webContentTitle = "WC WebContent Title 2",
-			webContentWebContent = "WC WebContent Title 1");
+			WebContent.addWithStructureCP(
+				structureName = "Structure Name B",
+				templateName = "Template Name B",
+				webContentTitle = "WC WebContent Title 2",
+				webContentWebContent = "WC WebContent Title 1");
 
-		PortletEntry.publish();
+			PortletEntry.publish();
+		}
 
-		Navigator.gotoStagedSitePage(
-			pageName = "Page A",
-			siteName = "Site Name");
+		task ("And: Add a WCD widget on the first page and select the first WC on it") {
+			Navigator.gotoStagedSitePage(
+				pageName = "Page A",
+				siteName = "Site Name");
 
-		Portlet.addPG(portletName = "Web Content Display");
+			Portlet.addPG(portletName = "Web Content Display");
 
-		WebContentDisplayPortlet.selectWebContent(webContentTitle = "WC WebContent Title 1");
+			WebContentDisplayPortlet.selectWebContent(webContentTitle = "WC WebContent Title 1");
+		}
 
-		Navigator.gotoStagedSitePage(
-			pageName = "Page B",
-			siteName = "Site Name");
+		task ("And: Add a WCD widget on the second page and select the second WC on it") {
+			Navigator.gotoStagedSitePage(
+				pageName = "Page B",
+				siteName = "Site Name");
 
-		Portlet.addPG(portletName = "Web Content Display");
+			Portlet.addPG(portletName = "Web Content Display");
 
-		WebContentDisplayPortlet.selectWebContent(webContentTitle = "WC WebContent Title 2");
+			WebContentDisplayPortlet.selectWebContent(webContentTitle = "WC WebContent Title 2");
+		}
 
-		Staging.openStagingAdmin(siteURLKey = "site-name-staging");
+		task ("When: User publish staging then exports the live site") {
+			Staging.openStagingAdmin(siteURLKey = "site-name-staging");
 
-		Staging.publishCustomPublication(rangeAll = "true");
+			Staging.publishCustomPublication(rangeAll = "true");
 
-		Navigator.gotoSitePage(
-			pageName = "Page A",
-			siteName = "Site Name");
+			Navigator.gotoSitePage(
+				pageName = "Page A",
+				siteName = "Site Name");
 
-		WebContentDisplayPortlet.viewContent(
-			displayText = "Link to Page",
-			pageName = "Page B",
-			webContentTitle = "WC WebContent Title 1");
+			WebContentDisplayPortlet.viewContent(
+				displayText = "Link to Page",
+				pageName = "Page B",
+				webContentTitle = "WC WebContent Title 1");
 
-		LAR.exportSiteCP(siteScopeName = "Site Name");
+			LAR.exportSiteCP(siteScopeName = "Site Name");
 
-		var key_larFileName = "${larFileName}";
-		var larFileName = LAR.getLarFileName();
+			var key_larFileName = "${larFileName}";
+			var larFileName = LAR.getLarFileName();
 
-		LAR.downloadLar();
+			LAR.downloadLar();
+		}
 
-		JSONGroup.deleteGroupByName(groupName = "Site Name");
+		task ("Then: User can import the second page with a linked WCD on a fresh site") {
```

Comment:
The current statement is a little confused. I think we could just say "Then: Verify that the import process is successful on a fresh site"